### PR TITLE
[Backport 1.2] added fix for gantt-charts test for different timezones

### PR DESCRIPTION
**Pull request by sumukhswamy**
### Description

Fix gantt-charts test for different timezones

### Issues Resolved

[List any issues this PR will resolve]

### Check List

- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: cypress/integration/plugins/gantt-chart-dashboards/gantt_ui.spec.js [hunk 1]
```diff
@@ -5,8 +5,12 @@
 
 /// <reference types="cypress" />
 
+import customParseFormat from 'dayjs/plugin/customParseFormat';
+import dayjs from 'dayjs';
```
Comment (kavilla) [inline cypress/integration/plugins/gantt-chart-dashboards/gantt_ui.spec.js:9]: I don't see this installed is it included in another dependency?

Comment (ps48) [reply cypress/integration/plugins/gantt-chart-dashboards/gantt_ui.spec.js:9]: True, We'll update the PR to use moment as mentioned here: https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/515. Thanks for the catch.

Comment (sumukhswamy) [reply cypress/integration/plugins/gantt-chart-dashboards/gantt_ui.spec.js]: Has been updated

Comment (ps48) [reply cypress/integration/plugins/gantt-chart-dashboards/gantt_ui.spec.js]: @kavilla we updated both the PRs to use `moment js`

**Review by kavilla — APPROVED** [review]

**Pull request merged**